Sources/SwiftInspectorVisitors/Tests/EnumVisitorSpec.swift

```diff
@@ -1,398 +0,0 @@
-                && $0.parentType?.asSource == "FooEnum"
-            }
-            expect(matching.count) == 1
-          }
-
-          it("finds FooEnum.FooFooClass.BarFooFooEnum") {
-            let matching = self.sut.enums.filter {
-              $0.name == "BarFooFooEnum"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooEnum.FooFooClass"
-            }
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level enums") {
-          it("asserts") {
-            let content = """
-            public enum FooEnum {}
-            public enum BarEnum {}
-            """
-
-            // The EnumVisitor is only meant to be used over a single enum.
-            // Using a EnumVisitor over a block that has multiple top-level
-            // classes is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level struct after a top-level enum") {
-          it("asserts") {
-            let content = """
-            public enum FooEnum {}
-            public struct FooStruct {}
-            """
-
-            // The EnumVisitor is only meant to be used over a single enum.
-            // Using a EnumVisitor over a block that has a top-level struct
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-
-        context("with a top-level class after a top-level enum") {
-          it("asserts") {
-            let content = """
-            public enum FooEnum {}
-            public class FooClass {}
-            """
-
-            // The EnumVisitor is only meant to be used over a single enum.
-            // Using a EnumVisitor over a block that has a top-level class
-            // is API misuse.
-            expect(try VisitorExecutor.walkVisitor(
-                    self.sut,
-                    overContent: content))
-              .to(throwAssertion())
-          }
-        }
-      }
-
-      context("visiting a code block with a top-level struct declaration") {
-        it("asserts") {
-          let content = """
-            public struct FooStruct {}
-            """
-
-          // The EnumVisitor is only meant to be used over a single enum.
-          // Using a EnumVisitor over a block that has a top-level struct
-          // is API misuse.
-          expect(try VisitorExecutor.walkVisitor(
-                  self.sut,
-                  overContent: content))
-            .to(throwAssertion())
-        }
-      }
-
-      context("visiting a code block with a top-level class declaration") {
```

Comment:
this test was not moved since a `NestableDeclarationVisitor` can parse this declaration type